docs/userguides/publishing.md

```diff
@@ -0,0 +1,33 @@
+# Publishing
```

Comment:
Purposely not including this in `index.md` until publishing, as a feature, is complete

Reply:
Done!